app/scss/editQuestion.scss

```diff
@@ -0,0 +1,114 @@
+@import 'imports/variables';
+
+* {
```

Comment:
dont need to nest everything inside this, have this separate

Reply:
correct - think it was just a typo that got out of hand. fixed